### PR TITLE
Add a workflow for scanning python deps

### PR DESCRIPTION
**Pull request by pmaslana**

### CODE REVIEW

File: .github/workflows/snyk-python-scan.yml
Comment (cmmarslender) [inline .github/workflows/snyk-python-scan.yml:2]:
```suggestion
name: 🚨 Snyk Python Scan
```

File: .github/workflows/snyk-python-scan.yml
Comment (cmmarslender) [inline .github/workflows/snyk-python-scan.yml:12]: Why not cancel in progress runs on the same branch like we do everywhere else in this repo?

**Review by cmmarslender — CHANGES_REQUESTED** [review]

**Review by cmmarslender — DISMISSED** [review]

**Review by emlowe — DISMISSED** [review]

File: .github/workflows/snyk-python-scan.yml
Comment (altendky) [inline .github/workflows/snyk-python-scan.yml:6]: @pmaslana, should this follow the standard pattern (plus cron)?

https://github.com/Chia-Network/chia-blockchain/blob/939b129f9f72dc3ee2aea0498c22363fac16e141/.github/workflows/test.yml#L3-L13

File: .github/workflows/snyk-python-scan.yml
Comment (altendky) [inline .github/workflows/snyk-python-scan.yml:13]: Remove extra leading space.

```suggestion
on:
  push:
    branches:
    - long_lived/**
    - main
    - release/**
    tags:
    - '**'
  pull_request:
    branches:
    - '**'
  workflow_dispatch: null
```

**Review by altendky — APPROVED** [review]
If this was waiting on my approval, my apologies for letting it sit around so long.  `:[`  Might be worth a daily scheduled run?  We generally run _something_ more often than that anyways so perhaps it is pointless, but it's also probably minimal extra load too.

I haven't looked through the results of this or checked to make sure it triggers emails or...  If I need to review that for this as well, let me know.

**Review by emlowe — APPROVED** [review]

**Pull request merged**
